src/components/MediaPlayer/VideoJS/components/js/VideoJSProgress.js

```diff
@@ -184,6 +186,10 @@ function ProgressBar({ player, handleTimeUpdate, times, options }) {
 
   const isMultiSourced = options.targets.length > 1 ? true : false;
 
+  let initTimeRef = React.useRef(initCurrentTime);
+  const setInitTime = (t) => {
+    initTimeRef.current = 0;
```

Comment:
Should this set to t instead of 0?
```suggestion
    initTimeRef.current = t;
```